### PR TITLE
chore: replace broken link with new discord icon source

### LINKED ISSUE #1990: [DOCS] fix discord icon broken link in readme
### Issue Description

The issue involved replacing a broken link in the README with a new icon source to maintain the documentation's integrity and functionality.

### Screenshots or Examples (if applicable)

linkshub

### Proposed Solution (optional)

I have fixed it. Once I am assigned, I will open a PR.

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+)
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by oyepriyansh**
## Fixes Issue
Closes #1990

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
Looks good to me

**Comment by Anmol-Baranwal** [comment]
| Header | Header |
|--------|--------|
| image | image | 

I think the first one looks clean and better.

**Comment by CBID2** [comment]
> | Header | Header |
> 
> |--------|--------|
> 
> | image | image | 
> 
> 
> 
> I think the first one looks clean and better.
> 
> 
> 
> @CBID2 @rupali-codes 
> 
> What do you think?
> 
> 
> 
> He has included the second one in the readme.

The second one does look cleaner @Anmol-Baranwal

**Comment by oyepriyansh** [comment]
I can do any

**Comment by Anmol-Baranwal** [comment]
> The second one does look cleaner @Anmol-Baranwal

Okay, then we will go with the second one (current version)

File: README.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Visit https://linkshub.dev to see the live demo!
 
 Connect with us for future updates
 
-- Join the LinksHub community on [Discord](https://discord.com/invite/NvK67YnJX5) <img src="https://github.com/shelar1423/DocsGPT/assets/82649533/bbd74228-f41f-4194-8fe2-531f7b29d61f" alt="Discord" width="20" height="20"  align="center"> .
+- Join the LinksHub community on [Discord](https://discord.com/invite/NvK67YnJX5) <img src="https://oyepriyansh.pages.dev/assets/github/readme/discord.svg" alt="Discord" width="17" height="17"  align="center"> .
```
Comment (Anmol-Baranwal) [inline README.md:80]:
```suggestion
- Join the LinksHub community on [Discord](https://discord.com/invite/NvK67YnJX5) <img src="https://github.com/rupali-codes/LinksHub/assets/74038190/0a9bece0-8378-4620-bcf5-0f6e4b1aa6bd" alt="Discord" width="20" height="20"  align="center">.
```

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
It is always better to use relative links to the repository instead of direct URLs. In this way, even if the original link is removed, there will be no issues.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Review by CBID2 — APPROVED** [review]
Looks good to me

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**